dm/worker/server.go

```diff
@@ -142,16 +146,21 @@ func (s *Server) Start() error {
 		defer s.wg.Done()
 		// TODO: handle fatal error from observeSourceBound
 		//nolint:errcheck
-		s.observeSourceBound(s.ctx, revBound)
+		for {
+			s.observeSourceBound(s.ctx, revBound)
+			s.stopKeepAlive()
+			select {
+			case <-s.ctx.Done():
+				return
+			case <-time.After(keepaliveTimeout * 2):
```

Comment:
we didn't give every variables a proper name 😂 `keepaliveTimeout` is not as you expected. It should be `s.cfg.KeepAliveTTL`

also welcome to give old variables better names